samples/tfm_integration/psa_crypto/prj.conf

```diff
@@ -11,6 +11,7 @@ CONFIG_LOG_DEFAULT_LEVEL=4
 #CONFIG_PSA_SHELL=y
 
 CONFIG_BUILD_WITH_TFM=y
+CONFIG_TFM_IPC=y
```

Comment:
@urutva @microbuilder  guess if we do that, then the only sample we'll be testing without IPC is the PS-sample. Just FYI :)